certified-connectors/Cognizant Automation Center/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1105 @@
+    "contact": {
+      "name": "AutomationCenter",
+      "url": "https://automationcenter.cognizantgoc.com/portal/boot/signon#",
+      "email": "AutomationCenter@cognizant.com"
+    }
+  },
+  "host": "automationcenter.cognizantgoc.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/portal/ino/api/v3/collab/activity/{ActivityID}/actions/{ActivityActionID}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "Success": {
+                  "type": "boolean",
+                  "description": "Success"
+                },
+                "DateTZConversionColumn": {
+                  "type": "array",
+                  "items": {},
+                  "description": "DateTZConversionColumn"
+                },
+                "Data": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "Status": {
+                        "type": "string",
+                        "description": "Status"
+                      },
+                      "Automation Status": {
+                        "type": "string",
+                        "description": "Automation Status"
+                      },
+                      "Status Id": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "Status Id"
+                      },
+                      "Automation Status Id": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "Automation Status Id"
+                      },
+                      "Output Status Id": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "Output Status Id"
+                      }
+                    }
+                  },
+                  "description": "Data"
+                },
+                "TotalCount": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "TotalCount"
+                },
+                "ColumnMapping": {
+                  "type": "array",
+                  "items": {},
+                  "description": "ColumnMapping"

```

Comment:
@SelvaGanesh-Cognizant All the description values of properties in the swagger are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

Reply:
@sriyen-msft Update the description of Supported operations. Response payload is generated and updated using swagger UI using import template operation. So response Key name and description remains the same. Kindly let me know this will create an impact.

Reply:
@SelvaGanesh-Cognizant Yes, all he description values of properties in the swagger are expected to be free from grammatical and spell errors.